keras_hub/src/models/stable_diffusion_3/mmdit.py

```diff
@@ -595,9 +595,30 @@ def build(self, inputs_shape, context_shape, timestep_embedding_shape):
         self.context_block.build(context_shape, timestep_embedding_shape)
 
     def _compute_attention(self, query, key, value):
+        batch_size = ops.shape(query)[0]
+
+        # Use the fast path when `ops.dot_product_attention` and flash attention
+        # are available.
+        if (
+            hasattr(ops, "dot_product_attention")
+            and hasattr(keras.config, "is_flash_attention_enabled")
+            and keras.backend.backend() != "tensorflow"
```

Comment:
Sure!

I have submitted a PR to core Keras:
https://github.com/keras-team/keras/pull/20615
With that change, the cost time of the tensorflow will be comparable to jax (w/o flash attention)
- tensorflow: 10.57s
- jax: 10.61s

Reply:
> Should we test this somehow?

I’m not sure how to test this since `ops.dot_product_attention` is intended to be a drop-in replacement.
Should I compare the numeric w/ and w/o `ops.dot_product_attention`?

Reply:
Thanks! https://github.com/keras-team/keras/pull/20615, yeah I think that's the way to go.

Hmm, yeah as long as the code is being exercised, probably fine to leave as is. Let's go with this!